openedx/features/enterprise_support/tests/test_api.py

```diff
@@ -364,8 +364,6 @@ def test_data_consent_required(self, mock_get_consent_url, mock_consent_necessar
         self.check_data_sharing_consent(consent_required=True, consent_url=consent_url)
 
         mock_get_consent_url.assert_called_once()
-        mock_enterprise_enabled.assert_called_once()
-        mock_consent_necessary.assert_called_once()
```

Comment:
Invalid assertion names for mock 1.0.1, and these aren't called reliably depending on the sequence of prior tests and what code paths they initialize.